app/services/doc_auth/errors.rb

```diff
@@ -123,7 +123,7 @@ module Errors
       SELFIE_FAILURE => { long_msg: SELFIE_FAILURE, field_msg: SELFIE_FAILURE, hints: false },
       SELFIE_NOT_LIVE => { long_msg: SELFIE_NOT_LIVE, field_msg: SELFIE_FAILURE, hints: false },
       SELFIE_POOR_QUALITY => { long_msg: SELFIE_POOR_QUALITY, field_msg: SELFIE_FAILURE, hints: false },
-    }
+    }.each { |_k, v| v.freeze }.freeze
```

Comment:
ok so I realized another way to do this same thing, sorry for the duplicate suggestion
```suggestion
    }.transform_values(&:freeze).freeze
```